### PR TITLE
Dev xmlplugin merge schema validation

### PR DESCRIPTION
**Pull request by jan-vcapgemini**
Adresses #173.

Implements

* various fixes of unit test resource files
* local xsd schemas for facelets xml validation
* possibility to provide a custom merge schema folder location (taken from templates)

Requires:  #1204

@devonfw/cobigen

### CODE REVIEW

File: cobigen/cobigen-xmlplugin/pom.xml [hunk 1]
```diff
@@ -21,18 +21,18 @@
...
       <groupId>com.devonfw.cobigen</groupId>
       <artifactId>core-api</artifactId>
-      <version>6.1.0-SNAPSHOT</version>
+      <version>6.2.0-SNAPSHOT</version>
     </dependency>
     <dependency>
       <artifactId>lexeme</artifactId>
       <groupId>com.github.maybeec</groupId>
-      <version>1.0.0</version>
+      <version>1.1.0-SNAPSHOT</version>
```
Comment (maybeec) [inline cobigen/cobigen-xmlplugin/pom.xml:29]: change if lexeme released

File: cobigen/cobigen-xmlplugin/pom.xml [hunk 2]
```diff
@@ -21,18 +21,18 @@
     <dependency>
       <groupId>com.devonfw.cobigen</groupId>
       <artifactId>core-api</artifactId>
-      <version>6.1.0-SNAPSHOT</version>
+      <version>6.2.0-SNAPSHOT</version>
```
Comment (maybeec) [inline cobigen/cobigen-xmlplugin/pom.xml:24]: I think core has been released, please check with @MikeSchumacherCapgemini  or on maven repository

File: cobigen/cobigen-xmlplugin/pom.xml (hunk 1 above)
Comment (jan-vcapgemini) [reply cobigen/cobigen-xmlplugin/pom.xml]: adjusted.

File: cobigen/cobigen-xmlplugin/pom.xml (hunk 2 above)
Comment (jan-vcapgemini) [reply cobigen/cobigen-xmlplugin/pom.xml:24]: no change required.

**Review by maybeec — APPROVED** [review]

**Comment by maybeec** [comment]
@jan-vcapgemini could you please look at the build? It comes with compilation errors.

**Review by maybeec — APPROVED** [review]

**Pull request merged**